proposals/tools.md

```diff
@@ -0,0 +1,90 @@
+# OCI Runtime-Tools and Image-Tools Project Proposals
+
+## Abstract
+The below proposal contemplates creation of two tools projects (runtime-tools and image-tools.)
+These would be associated with the OCI specification projects (runtime-spec and image-spec) and serve as repositories for testing  tools. 
+
+## Background
+Testing tools have been organically developed by the OCI developer community.
+These include:
+* Runtime-spec test code in a OCITools repository
+* Image-spec test code in the image-spec repository
+
+On Monday, August 22nd, 2016, the OCI Certification Program WG designated Chris Aniszczyk and Rob Dolin to draft a proposal for formalizing approval of tools.
+
+On Wednesday, August 24th, 2016, the OCI Developer Community (TDC) met and recommended establishing a tools repository corresponding to each of the spec repositories.  
+
+## Proposal
+With repositories under the http://github.com/OpenContainers organization:
+* Rename the OCITools repository as runtime-tools
+* Create a new repository named image-tools and populate this with the files from: https://github.com/opencontainers/image-spec/tree/master/cmd/oci-image-tool
+
+### Project Descriptions
+Below are brief draft project descriptions intended to serve as an initial abstract for each project:
+* Runtime-Tools: Tools for testing of container runtimes implementing the OCI runtime-spec.  
+This includes code that tests a runtime's conformance to the OCI Container Runtime spec.
+* Image-Tools: Tools for testing of container images implementing the OCI image-spec.  
+This includes code that validates a file's conformance to the OCI Container Image Format spec.  
+
+### Initial Maintainers
```

Comment:
On Wed, Aug 31, 2016 at 03:54:52PM -0700, Michael Crosby wrote:
> I think we have pretty clear rules around how maintainers are added.
> The initial set is usually the existing maintainers since these new
> projects have the same scope.  Additional maintainers should be
> separate votes based on the individuals

This makes sense to me, and saves the TOB some thinking.